### DISCUSSION TITLE
[2021.10.19] 중성화 여부 파악하기

### DISCUSSION THREAD

**Discussion by leechang4** [pyunji/Hyundai-ITnE-Team_QuadCore#145] [SQL]
[중성화 여부 파악하기](https://programmers.co.kr/learn/courses/30/lessons/59409)

**Comment by pyunji** [comment]
[중성화 여부 파악하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59049.sql)

**Comment by hyk339** [comment]
[중성화 여부 파악하기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59409.sql)

**Comment by chaeyeon4u** [comment]
[중성화 여부 파악하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59409.sql)

**Comment by leechang4** [comment]
[중성화 여부 파악하기 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR%2059049.sql)
